app/ui/latest/src/test/java/com/mudita/mail/ui/usecase/signIn/viewModel/SignInViewModelTest.kt

```diff
@@ -62,4 +62,24 @@ class SignInViewModelTest {
         verify(exactly = 1) { interactor.getProviders() }
         assertEquals(expectedList, viewModel.uiState.value.providers)
     }
+
+    @Test
+    fun `viewModel's selection of outlook should result in navigator call with outlook provider type`() {
+        val expectedProvider = ProviderType.OUTLOOK
+        every { navigator.moveToEmailScreen(expectedProvider) } returns Unit
+
+        SignInViewModel(interactor, navigator).selectProvider(expectedProvider)
+
+        verify(exactly = 1) { navigator.moveToEmailScreen(expectedProvider) }
+    }
+
+    @Test
+    fun `viewModel's selection of gmail should result in navigator call with gmail provider type`() {
+        val expectedProvider = ProviderType.GMAIL
+        every { navigator.moveToEmailScreen(expectedProvider) } returns Unit
```

Comment:
there is a simple notation in mockK for this.
Instead of `returns Unit` you can use `just Runs`